CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -82,7 +82,6 @@ public function preProcess() {
 
     parent::preProcess();
     $this->_id = CRM_Utils_Request::retrieve('id', 'Positive', $this, TRUE);
-    // @todo don't set this - rely on parent $this->contactID
     $this->_contactId = CRM_Utils_Request::retrieve('cid', 'Positive', $this, TRUE);
```

Comment:
@eileenmcnaughton we cannot remove this as $_GET['cid'] isn't available at parent instance.